doctrine/reverse_engineering.rst

```diff
@@ -70,6 +70,22 @@ files: ``BlogPost.php`` and ``BlogComment.php``.
 
         $ php bin/console doctrine:mapping:import 'App\Entity' xml --path=config/doctrine
 
+    In this case, make sure to adapt your mapping configuration accordingly:
+
+    .. code-block:: yaml
+
+        doctrine:
```

Comment:
Can you add the `# config/packages/doctrine.yaml` filename comment above this line?